.buildkite/pipeline.yml

```diff
@@ -1,26 +1,32 @@
 x-common-params:
   - &gb-mobile-docker-container
     docker#v3.8.0:
-      image: "public.ecr.aws/automattic/gb-mobile-image:latest"
+      image: 'public.ecr.aws/automattic/gb-mobile-image:latest'
```

Comment:
I think VSCode this this for me. Happy to revert if considered noisy. At some point, we might want to establish a standard YAML syntax (or maybe WordPress.org already has one that we can adopt?) but this doesn't seem the time.

Reply:
My guess is that Prettier may have done this, if you have the VS Code plugin installed. 

I believe there are a few files in `gutenberg-mobile` and `gutenberg` — e.g. release notes — that do not embrace WordPress coding standards or leverage automated formatting. There is opportunity for us to further embracing those standards and configure the repositories to enforce them via Prettier or eslint.

Reply:
Thanks for the pointer @dcalhoun